### PR TITLE
Fix missing telemetry nodes

### PR DESCRIPTION
**Pull request by gmourier**

### CODE REVIEW

File: learn/what_is_meilisearch/telemetry.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ We use two data-collection tools: [Amplitude](https://amplitude.com/amplitude-an
 - Amount of time the MeiliSearch instance has been running
 - MeiliSearch version
 - Technical specs of the device running MeiliSearch
-- Number of documents per index
+- Database size per instance
+- Number of documents per instance
```
Comment (guimachiavelli) [inline learn/what_is_meilisearch/telemetry.md:21]: According to @curquiza's PR #1065, `Number of documents per index` will only be available on v0.22.

Is `Number of documents per instance` available on v0.21? Or is the proposed addition only correcting exchanging `index` with `instance` but refer to the same field?

Comment (gmourier) [reply learn/what_is_meilisearch/telemetry.md:21]: Hey @guimachiavelli!

An instance can contain several indexes.

So the @curquiza fix is ok! In 0.21 we already have `Number of documents per instance`.

Comment (guimachiavelli) [reply learn/what_is_meilisearch/telemetry.md:21]: Got it, thanks for clarifying!

**Review by guimachiavelli — APPROVED** [review]
Looks great, thanks for this! 🧨

**Pull request merged**